stripe/util.py

```diff
@@ -1,10 +1,24 @@
+from __future__ import print_function
+
 import logging
 import sys
 import os
+import stripe
```

Comment:
Nit, but per [PEP 8](https://www.python.org/dev/peps/pep-0008/#imports), local imports should come after standard library imports.